scss/_custom-variables.scss

```diff
@@ -228,6 +228,11 @@ $spacers: map-merge(
 $badge-pill-radius: .25rem;
 
 // >> Breadcrumbs
+$breadcrumb-font-size: small;
```

Comment:
The previous suggestion:
“The variable value overrides removed from `scss/_custom-variables.scss` to shut up the linting check for unused variables can go back, because they're now used somewhere.”
is contingent on them _actually_ being used, so they can't be added without modifying `scss/_pagination.scss`